src/CbcCInterface.jl

```diff
@@ -138,60 +140,60 @@ function loadProblem(prob::CbcModel,
 
     @cbc_ccall loadProblem Cvoid (Ptr{Cvoid}, Int32, Int32, Ptr{CoinBigIndex},
         Ptr{Int32}, Ptr{Float64}, Ptr{Float64}, Ptr{Float64},
-        Ptr{Float64}, Ptr{Float64}, Ptr{Float64}) prob.p ncol nrow mat.colptr.-Int32(1) mat.rowval.-Int32(1) mat.nzval vec_or_null(Float64, col_lb, ncol) vec_or_null(Float64, col_ub, ncol) vec_or_null(Float64, obj, ncol) vec_or_null(Float64, row_lb, nrow) vec_or_null(Float64, row_ub, nrow)
+        Ptr{Float64}, Ptr{Float64}, Ptr{Float64}) prob ncol nrow mat.colptr.-Int32(1) mat.rowval.-Int32(1) mat.nzval vec_or_null(Float64, col_lb, ncol) vec_or_null(Float64, col_ub, ncol) vec_or_null(Float64, obj, ncol) vec_or_null(Float64, row_lb, nrow) vec_or_null(Float64, row_ub, nrow)
```

Comment:
What about `mat.nzval`?

Reply:
I think this is not a problem, because having `ccall` just keep `mat.nzval` (a `Vector{Float64}`) alive is already sufficient. It would be a problem if this were `pointer(mat.nzval)`.

Reply:
Got it